### ISSUE TITLE
Allow users of Emacs to play with Spacemacs without touching their configuration

### ISSUE THREAD

**Issue by DamienCassou** [syl20bnr/spacemacs#4850]
Currently, trying Spacemacs requires backuping `~/.emacs.d` and replacing it with Spacemacs' own init file. If you want to attract current Emacs users, that's not really reasonable because they might want to keep Emacs running while trying Spacemacs. I suggest you add to the documentation some lines about running Spacemacs in parallel. On IRC, someone suggested

```
$ mkdir ~/spacemacs/
$ git clone https://github.com/syl20bnr/spacemacs ~/spacemacs/.emacs.d
$ HOME=~/spacemacs emacs
```

**Comment by StreakyCobra** [comment]
Indeed :+1:

**Comment by ghoot** [comment]
+1

In addition this method can be used to test and run in parallel other configs e.g. Purcell, Bling, Prelude etc

I also use this method to keep a separate spacemacs clone with untouched config, to quickly test against a "vanilla" spacemacs whenever I come across potential issues.

On OSX I wrap this up using Automator to run something like

`env HOME=$HOME/spacemacs-vanilla/ open -n /Applications/Emacs.app`

which is then saved into your Applications folder and is easy to trigger through Alfred/Spotlight.

Of course you can do the same with scripts to manage symlinks etc but this is much more convenient if you are doing it regularly.

It's not unusual for me to have an instance of my previous emacs config, an instance of spacemacs vanilla, and an instance of my current spacemacs config, all open and running in parallel.

**Comment by kolewu** [comment]
Maybe I have found a better way than symlinking `.emacs.d` or changing `HOME`, that mimics the behaviour of the original init-code from `setup.el`.  I have described it in detail on emacs.SE: http://emacs.stackexchange.com/a/20508/934

**Comment by StreakyCobra** [comment]
@kolewu You can create a new PR modifying the FAQ entry in order to add a brief explanation of your solution with a link pointing to SE :-)

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
